atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueue.java

```diff
@@ -137,9 +137,9 @@ public long sweepNextBatch(ShardAndStrategy shardStrategy, long sweepTs) {
 
         SweepBatchWithPartitionInfo batchWithInfo = reader.getNextBatchToSweep(shardStrategy, lastSweptTs, sweepTs);
         SweepBatch sweepBatch = batchWithInfo.sweepBatch();
-        metrics.registerEntriesReadInBatch(shardStrategy, sweepBatch.entriesRead());
-
+        // todo(gmaretic): update last seen commit ts here
         deleter.sweep(sweepBatch.writes(), Sweeper.of(shardStrategy));
+        metrics.registerEntriesReadInBatch(shardStrategy, sweepBatch.entriesRead());
```

Comment:
this is a driveby that will greatly improve metrics -- we can now match entries read with entries written even when deletes fail